src/main/java/nextstep/subway/auth/ui/token/TokenAuthenticationInterceptor.java

```diff
@@ -28,8 +30,13 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
         AuthenticationToken authenticationToken = convert(request);
         Authentication authentication = authenticate(authenticationToken);
 
-        // TODO: authentication으로 TokenResponse 추출하기
         TokenResponse tokenResponse = null;
+        if (authentication == null) {
+            throw new RuntimeException();
+        }
+        String payload = new ObjectMapper().writeValueAsString(authentication.getPrincipal());
+        String token = jwtTokenProvider.createToken(payload);
+        tokenResponse = new TokenResponse(token);
```

Comment:
```suggestion
        if (authentication == null) {
            throw new RuntimeException();
        }
        String payload = new ObjectMapper().writeValueAsString(authentication.getPrincipal());
        String token = jwtTokenProvider.createToken(payload);
        TokenResponse tokenResponse = new TokenResponse(token);
```

뼈대코드는 마음껏 수정이 가능하십니다!